devdoc/Releasing_Tycho.md

```diff
@@ -22,44 +47,19 @@ This page describes the steps necessary to create releases of the Tycho project.
 </settings>
 </source>
 ```
-- [ ] Checkout the tag version `tycho-<VERSION>` so current commit is the tag.
+
+- [ ] Checkout branch `tycho-N.M.x`
+- [ ] Update version to remove `-SNAPSHOT` with `mvn org.eclipse.tycho:tycho-versions-plugin:set-version -DnewVersion=<VERSION>`
+- [ ] Update versions in tycho-demo folder
 - [ ] Sync to release commit and deploy to nexus staging repository: `mvn clean deploy -Prelease -DforgeReleaseId=sonatype-nexus-staging -DforgeReleaseUrl=https://oss.sonatype.org/service/local/staging/deploy/maven2/`
-- [ ] In parallel of previous <tt>mvn</tt> command, prepare update [tycho-demo projects](http://git.eclipse.org/c/tycho/org.eclipse.tycho-demo.git) to the latest release and push the change as a review to Gerrit
-- [ ] When previous `mvn` command is complete, generate site docs using `mvn install site site:stage -DskipTests` and check the result from `target/staging` seems viable.
 - [ ] On https://oss.sonatype.org/#stagingRepositories , Close the staging repository, get the staging repo URL from Nexus
-- [ ] Update [release notes](https://github.com/eclipse/tycho/blob/master/RELEASE_NOTES.md) to replace reference to snapshot build to reference to staging repo
-- [ ] Announce the temporary stage URL on tycho-user@eclipse.org:
-```
-Subject: please test staged tycho <VERSION>
-
-Tycho release <VERSION&> has been staged. For details of new features and bugfixes, see release notes [1].
-Please help by testing the staged build. To use it, change your tycho version to <VERSION> and add snippet [2] to your pom.
-
-We plan to promote this release in one week unless major regressions are found.
-
-Regards,
-Tycho team
-
-[1] https://github.com/eclipse/wildwebdeveloper/blob/master/RELEASE_NOTES.md
-[2]
-  <pluginRepositories>
-    <pluginRepository>
-      <id>tycho-staged</id>
-      <url><NEXUS_OSS_STAGING_URL></url
-    </pluginRepository>
-  </pluginRepositories>
-</pre>
-```
-- [ ] Prepare documentation on  the webite, using git repo https://git.eclipse.org/c/www.eclipse.org/tycho.git/ : copy the local site docs folders `target/staging/sitedocs` to the existing `sitedocs` folder and then submit as Gerrit review (don't merge yet)
-
-... Wait until review date (usually a week later)...
-
-- [ ] After ~1 week of testing, promote the stage repository on https://oss.sonatype.org/
 - [ ] Wait for artifacts to be available on Maven central.
-- [ ] Push the release tags to git: `git push origin --tags`
-- [ ] Update https://github.com/eclipse/tycho/blob/master/RELEASE_NOTES.md to remove reference to staging repo
-- [ ] Merge the update to `org.eclipse.tycho-demo` and website in Gerrit
-- [ ] Announce the release
+- `git tag <TYCHO_VERSION>`
```

Comment:
maybe tag before sync to nexus ?

Reply:
is it synchronize with github release?  Where do you feed the content of the release in github? RELEASE_NOTES.md is enough to feed it ?

Reply:
Maybe creating the tag/release [as described here](https://docs.github.com/en/github/administering-a-repository/releasing-projects-on-github/managing-releases-in-a-repository)?